src/object/icecrusher.cpp

```diff
@@ -82,10 +103,32 @@ IceCrusher::set_state(IceCrusherState state_, bool force)
       physic.enable_gravity (true);
       m_sprite->set_action("crushing");
       break;
+    case CRUSHING_RIGHT:
+      set_group(COLGROUP_MOVING_STATIC);
+	    physic.reset ();
+      physic.enable_gravity (false);
+      m_sprite->set_action("idle");
+	  break;
+	case CRUSHING_LEFT:
+      set_group(COLGROUP_MOVING_STATIC);
+	  physic.reset ();
```

Comment:
Please fix indentation throughout the code.

Reply:
I fixed what I saw but if there is more indentation issues let me know.